text-minimessage/src/main/java/net/kyori/adventure/text/minimessage/MiniMessage.java

```diff
@@ -98,23 +97,23 @@
    * <p>Useful for untrusted input.</p>
    *
    * @param input the input message, with tokens
-   * @param placeholders the placeholder resolver adding known placeholders
+   * @param tags a tag resolver to provide any additional tags that can be understood, combined with the overall tag resolver
    * @return the output, without tokens
    * @since 4.10.0
    */
-  @NotNull String stripTokens(final @NotNull String input, final @NotNull PlaceholderResolver placeholders);
+  @NotNull String stripTokens(final @NotNull String input, final @NotNull TagResolver tags);
 
   /**
-   * Deserializes a string into a component, with a placeholder resolver to parse placeholders of the form {@code <key>}.
+   * Deserializes a string into a component, with a tag resolver to parse tags of the form {@code <key>}.
    *
-   * <p>Placeholders will be resolved from this resolver before the resolver provided in the builder is used.</p>
+   * <p>Tags will be resolved from the resolver parameter before the resolver provided in the builder is used.</p>
    *
    * @param input the input string
-   * @param placeholderResolver the placeholder resolver
+   * @param tagResolver the tag resolver for any additional tags to handle.
```

Comment:
This is much nicer than the docs for the same element of the `stripTokens` and `escapeTokens` methods.

```suggestion
   * @param tagResolver the tag resolver for any additional tags to handle
```